### ISSUE TITLE
it does work after P2 finished

### ISSUE THREAD

**Issue by conanlee88** [madMAx43v3r/chia-plotter#271]
when the P2 finished, the process seems to go down
![stop_in_P2](https://user-images.githubusercontent.com/25920722/121723596-49572180-cb19-11eb-9ff0-a5c696827f58.png)
![cpu_usage](https://user-images.githubusercontent.com/25920722/121723767-84f1eb80-cb19-11eb-9558-9fc6d622a87a.png)

**Comment by madMAx43v3r** [comment]
does it happen always, or just once?

**Comment by michaelforfxhelp** [comment]
> does it happen always, or just once?

sometimes, but if is happen, it will never recover.

**Comment by conanlee88** [comment]
> > does it happen always, or just once?
> 
> sometimes, but if is happen, it will never recover.

yeah, that is right, how do you fix it? make it run again?

**Comment by conanlee88** [comment]
> does it happen always, or just once?

once happened, it will happen again at different steps. sometimes at P2, sometimes at P3.

**Comment by michaelforfxhelp** [comment]
> > > does it happen always, or just once?
> > 
> > 
> > sometimes, but if is happen, it will never recover.
> 
> yeah, that is right, how do you fix it? make it run again?

![image](https://user-images.githubusercontent.com/13658345/121761349-c3ab9400-cb61-11eb-840e-1e8ca8f57be7.png)

Hey, my chia-plot run continue again!
1.the "[P2] Table 2 scan took 39989.7 sec" hung for a long night, I press enter when I wake up this morning, It run again.
2. then the "Wrote plot header with 268 bytes" continue for about half an hour，I did press Enter for many time but it show nothing. then I left it do nothing, when another half an hour pass, I work again ...

I don't know why. the 1 situation is more offen.

**Comment by michaelforfxhelp** [comment]
> > does it happen always, or just once?
> 
> once happened, it will happen again at different steps. sometimes at P2, sometimes at P3.

I used window version 0.0.3, and you ?

**Comment by conanlee88** [comment]
> > > does it happen always, or just once?
> > 
> > 
> > once happened, it will happen again at different steps. sometimes at P2, sometimes at P3.
> 
> I used window version 0.0.3, and you ?

i used linux, download source code, and build it local

**Comment by conanlee88** [comment]
> > > > does it happen always, or just once?
> > > 
> > > 
> > > sometimes, but if is happen, it will never recover.
> > 
> > 
> > yeah, that is right, how do you fix it? make it run again?
> 
> ![image](https://user-images.githubusercontent.com/13658345/121761349-c3ab9400-cb61-11eb-840e-1e8ca8f57be7.png)
> 
> Hey, my chia-plot run continue again!
> 1.the "[P2] Table 2 scan took 39989.7 sec" hung for a long night, I press enter when I wake up this morning, It run again.
> 2. then the "Wrote plot header with 268 bytes" continue for about half an hour，I did press Enter for many time but it show nothing. then I left it do nothing, when another half an hour pass, I work again ...
> 
> I don't know why. the 1 situation is more offen.

i shut down the process... and run it again, it will happen again. so you just keep the process, it will run after a long time?

**Comment by michaelforfxhelp** [comment]
> > > > > does it happen always, or just once?
> > > > 
> > > > 
> > > > sometimes, but if is happen, it will never recover.
> > > 
> > > 
> > > yeah, that is right, how do you fix it? make it run again?
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/13658345/121761349-c3ab9400-cb61-11eb-840e-1e8ca8f57be7.png)
> > Hey, my chia-plot run continue again!
> > 1.the "[P2] Table 2 scan took 39989.7 sec" hung for a long night, I press enter when I wake up this morning, It run again.
> > 2. then the "Wrote plot header with 268 bytes" continue for about half an hour，I did press Enter for many time but it show nothing. then I left it do nothing, when another half an hour pass, I work again ...
> > I don't know why. the 1 situation is more offen.
> 
> i shut down the process... and run it again, it will happen again. so you just keep the process, it will run after a long time?

Yes, I just press enter some times and keep the process. It looks takes 1026.78 sec for this pause section

**Comment by conanlee88** [comment]
next time，it will take such a long time again？or back normal？
> > > > > > does it happen always, or just once?
> 
> > > > > 
> 
> > > > > 
> 
> > > > > sometimes, but if is happen, it will never recover.
> 
> > > > 
> 
> > > > 
> 
> > > > yeah, that is right, how do you fix it? make it run again?
> 
> > > 
> 
> > > 
> 
> > > ![image](https://user-images.githubusercontent.com/13658345/121761349-c3ab9400-cb61-11eb-840e-1e8ca8f57be7.png)
> 
> > > Hey, my chia-plot run continue again!
> 
> > > 1.the "[P2] Table 2 scan took 39989.7 sec" hung for a long night, I press enter when I wake up this morning, It run again.
> 
> > > 2. then the "Wrote plot header with 268 bytes" continue for about half an hour，I did press Enter for many time but it show nothing. then I left it do nothing, when another half an hour pass, I work again ...
> 
> > > I don't know why. the 1 situation is more offen.
> 
> > 
> 
> > i shut down the process... and run it again, it will happen again. so you just keep the process, it will run after a long time?
> 
> 
> 
> Yes, I just press enter some times and keep the process. It looks takes 1026.78 sec for this pause section

**Comment by kacperapi** [comment]
I had the same problem  fixed it with few reinstalls, cleaning all disks and mounting/unmounting ram disk on top of reboot

**Comment by conanlee88** [comment]
> I had the same problem fixed it with few reinstalls, cleaning all disks and mounting/unmounting ram disk on top of reboot

i just clean the temp files and build a new temp folder， it will run again.  but once i run it with multi tasks,the first task will be done, but the second will be paused at any time. i think it may be something wrong with copy final file and plot together.

**Comment by conanlee88** [comment]
> does it happen always, or just once?

i find that the process does not shut down, just always in sleeping state.
